mno_analysis_tools/fetch_archives.py

```diff
@@ -0,0 +1,31 @@
+import argparse
+
+from core_data_modules.logging import Logger
+from rapid_pro_tools import RapidProClient
+
+log = Logger(__name__)
+log.set_project_name("FetchRawMessages")
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(description="Fetches archives from Rapid Pro instance")
+    parser.add_argument("domain", metavar="domain",
+        help="Domain that the instance of Rapid Pro is running on",
+    )
+    parser.add_argument("token", metavar="token",
+        help="Token for authenticating to the instance",
+    )
+    parser.add_argument("output_file", metavar="file", type=argparse.FileType(mode="w"),
```

Comment:
```suggestion
    parser.add_argument("output_file_path", metavar="output-file-path", type=argparse.FileType(mode="w"),
```

so that we're clear about which variables are paths vs file pointers.